Retrovizor.Domain/Repositories/Implementations/ClassRepository.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Retrovizor.Data.Entities;
+using Retrovizor.Data.Entities.Models;
+using Retrovizor.Domain.Repositories.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Retrovizor.Domain.Repositories.Implementations
+{
+    public class ClassRepository : IClassRepository
+    {
+        ClassRepository(RetrovizorContext context)
```

Comment:
Constructor nije public, ne znam jel to stvara problem ali bolje bit siguran. Ista stvar kod jos nekoliko klasa ali bolje da ne spamman isti komentar.

Reply:
Lapsus. Will fix.